tests/unit/opera/instance/test_attribute_mapping.py

```diff
@@ -97,20 +97,24 @@ def test_map_attribute_relationship_self(self, service_template):
 
         assert "steampunk" == relationship_instance.get_attribute(["SELF", "colour"])
 
-    @pytest.mark.parametrize("host",
-                             ["HOST", "my_node", "my_collector", "other"])
+    @pytest.mark.parametrize("host", [
+        ("HOST", "HOST as the 'host' attribute is not yet supported"),
+        ("my_node", "The 'host' attribute should be set to one of"),
+        ("my_collector", "The 'host' attribute should be set to one of"),
+        ("other", "The 'host' attribute should be set to one of")
+    ])
     def test_map_attribute_relationship_bad_host(self, service_template, host):
         node_source = service_template.find_node("my_collector")
         node_source_instance = next(iter(node_source.instances.values()))
         relationship_instance = next(iter(node_source_instance.out_edges["my_target"].values()))
 
-        with pytest.raises(DataError, match="Attribute host should be set to 'SELF'"):
+        with pytest.raises(DataError, match="The 'host' attribute should be set to one of"):
```

Comment:
There's no defined host TOSCA attribute, but there is a parameter host that should be set for some attribute.
I suggest: Attribute's 'host' should be set to one of... or Attribute's 'modelable_entity_name' should be set to one of

![image](https://user-images.githubusercontent.com/41727816/111272377-d54cd080-8632-11eb-9928-73f1a7eaf630.png)

Reply:
It seems I've misunderstood what I thought was a "host attribute", hah. Shows my lack of tosca. I've changed this to `The attribute's 'host'`.